### DISCUSSION TITLE
How to understand Next_Fire_Time

### DISCUSSION THREAD

**Discussion by HiradRoshandel** [quartznet/quartznet#1500] [Q&A]
I have connected my Quartz to a SQLServer and in the QRTZ_TRIGGERS table I see the values are set for NEXT_FIRE_TIME to something like 637789716000000000. What is this number? Is it unix time? I tried putting it in a epoch converter but it converts it to some date in March 1990. Can someone please help me understand this? Thanks

**Comment by lahma** [comment]
It's UTC ticks, see this explanation and query conversion: https://stackoverflow.com/questions/4225705/quartz-net-meaning-of-bigint-datetime
